persistence-api/src/main/java/io/stargate/db/schema/Column.java

```diff
@@ -414,6 +418,30 @@ public boolean isTuple() {
 
     Varint(14, BigInteger.class, false, "Arbitrary-precision integer"),
 
+    Point(
+        0, // custom type
+        io.stargate.db.schema.Point.class,
+        "org.apache.cassandra.db.marshal.PointType",
+        true,
+        null,
+        "Contains two coordinate values for latitude and longitude"),
+
+    Polygon(
+        0, // custom type
+        io.stargate.db.schema.Polygon.class,
+        "org.apache.cassandra.db.marshal.PolygonType",
+        true,
+        null,
+        "Contains three or more point values forming a polygon"),
+
+    LineString(
+        0, // custom type
+        io.stargate.db.schema.LineString.class,
+        "org.apache.cassandra.db.marshal.LineStringType",
+        true,
+        null,
+        "Contains two or more point values forming a line"),
+
```

Comment:
Maybe I'm nitpicking, but something is bothering me here: `Column.Type` is supposed to represent the "raw" type id in the protocol. If materializing a complete type requires additional parameters (that is, for anything that is not a primitive type), then we have a separate class. For example: `Column.Type.List` => `ListType`, etc.

Following that same pattern here, we should have a single `Column.Type.Custom`, and then a `CustomType implements Column.ColumnType` (that would wrap the marshal type name).

And then we would get _any_ custom type for free, because in `Conversion.getTypeFromInternal` you would be able do this:
```java
    Column.Type type = TYPE_MAPPINGS.get(abstractType.getClass());
    if (type == null) {
      type = new CustomType(abstractType.getClass().getName());
    }
```

Reply:
+1

Reply:
Good point. Thanks @olim7t ! I will certainly make a follow-up PR for this.

Reply:
Regarding "raw" types, though, I think they are raw in the sense of being non-parameterized, so the raw type of `List<Int>` is just `List`. Since geo types do not support parameterization, they are effectively "raw"... but I'll think about this some more.

Reply:
We seem to have a mix of protocol-level concepts (type IDs) and CQL-level concepts (CQL type names) :thinking:

Reply:
> Since geo types do not support parameterization

`Custom` is parameterized. Geo types are instances of that type for a particular class name, for example the point type is a `Custom("o.a.c.db.marshal.PointType")`.

> We seem to have a mix of protocol-level concepts (type IDs) and CQL-level concepts (CQL type names)

Yes, due to the fact that `Type` implements `ColumnType`. It's convenient because you can reuse the primitive type IDs to represent their respective primitive type. But IMHO it would be cleaner to keep the two concepts separated:
```java
enum Type {
  Ascii(1),
  Bigint(2),
  ...
}
class PrimitiveType implements ColumnType {
  private final Type typeId;
  ...
}
```
But anyway it's probably too late to change that, and definitely out of scope for this PR.